index.html

```diff
@@ -115,6 +115,42 @@ <h2>Guidelines</h2>
   </div>
 </div>
 
+<div class="p-strip--light is-bordered is-deep">
+  <div class="row">
+    <div class="col-12">
+      <h2>Downloads</h2>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-6">
+      <div class="p-heading-icon">
+        <div class="p-heading-icon__header">
+          <img class="p-heading-icon__img" src="https://assets.ubuntu.com/v1/a81f84ba-github-icon.svg">
+          <h3 class="p-heading-icon__title">Vanilla Framework</h3>
+        </div>
+        <p>Use our CSS framework to start building your project.</p>
+        <p>
+          <a class="p-link--strong" href="https://github.com/vanilla-framework/vanilla-framework">Download Vanilla Framework 1.6.3</a>
+          <p class="subtext u-no-margin--top">See the <a class="p-link--external" href="https://github.com/vanilla-framework/vanilla-framework/releases">release notes</a> for details on the latest updates.</p>
```

Comment:
We shouldn't be using bespoke styling on the Vanilla website. Can you remove and if this is something design wants they should file an issue.

Reply:
We have `small` which reduces the font-size or otherwise use `strong` to highlight the values. The issue with grey text is it can become inaccessible in some cases.